event_email_reminder/models/event.py

```diff
@@ -10,17 +11,28 @@
 class EventEvent(models.Model):
     _inherit = 'event.event'
 
+    @api.model
+    def _send_event_template(self, events, template, partner_ids):
+        ctx = self.env.context.copy()
+        ctx.update({
+            'events': events.sorted(lambda x: x.date_begin),
+        })
+        for partner_id in partner_ids:
+            template.with_context(ctx).send_mail(partner_id)
```

Comment:
A `lang` should be provided in context so the email is sent in the proper language:
```python
            ctx.update({
                'lang': partner_id.lang,
            })
```

Reply:
Please make yourself the change for finishing this.